### ISSUE TITLE
[FEATURE] required vars

### ISSUE THREAD

**Issue by dropje86** [jrasell/levant#63]
Just like with Terraform it would be nice to be able to set required vars:
```
variable foo {}
```
In yaml that would be an empty key:
```
---
foo:
```
This would make it easier to force a user setting a dynamic var like `version`

**Comment by jrasell** [comment]
So in this case it would mean a user would need to input the variable by typing on the CLI?

**Comment by dropje86** [comment]
Yeah either prompt or (in my case better) fail with a log message. We run Levant from CI so failing instead of waiting for user input would be nice.

Terraform has:
```
-input=true - Ask for input for variables if not directly set.
```
